pom.xml

```diff
@@ -313,8 +317,8 @@
                     </dependency>
                 </dependencies>
                 <configuration>
-                    <configLocation>swe_checkstyle.xml</configLocation>
-                    <suppressionsLocation>checkstyle-suppressions.xml</suppressionsLocation>
+                    <configLocation>${project.basedir}/swe_checkstyle.xml</configLocation>
```

Comment:
required as part of incrementalising (so I can consume this PR in claim plugin if this takes time to merge)